src/main/scala/com/codacy/pmd/DocGenerator.scala

```diff
@@ -288,6 +288,12 @@ object DocGenerator {
 
       val timeToFix = Patterns.timeToFix.get(patternId.value).map(Pattern.TimeToFix.apply)
 
+      val longDescriptionWithoutUnsupportedInfo = longDescription.trim
```

Comment:
Consider adding a comment with a link to the Jira issue to provide more context for people reading the code in the future. :+1: